app/models/user.rb

```diff
@@ -0,0 +1,26 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+  has_many :comments
+  has_many :likes
+  has_many :liked_posts, through: :likes, source: :post
+  has_many :liked_comments, through: :likes, source: :comment
+  validates :name, length: { maximum: 100 }, presence: true
+  validates :bio, length: { maximum: 300 }, allow_blank: true
+  has_one :picture
+
```

Comment:
- [ ] User model has a method that returns 3 most recent posts. Kindly make sure you create this method here ✍🏻

Reply:
Not required by the previous reviewer and the method already exist. In the requirements is not reflected that it needs to be executed when an instance of user is created plus it wouldn't make sense to run it since the user has just been created so it wouldn't return anything

Reply:
Well Implemented ✔️ 

![image](https://github.com/wickathou/rails-blog/assets/38088818/d88de21f-b7ce-4775-bd01-48921994ffea)

It is very much required!

![image](https://github.com/wickathou/rails-blog/assets/38088818/1f35a08e-4b36-420d-9298-68e4226ca2de)